semantic_release/settings.py

```diff
@@ -69,7 +69,7 @@ def _config_from_pyproject(path):
         return pyproject.get("tool", {}).get("semantic_release", {})
 
     except toml.TomlDecodeError:
-        logger.debug("Could not decode pyproject.toml")
```

Comment:
Does this just need the `as err` that you have in your example in the issue?

Reply:
Ah, yes! My apologies, I tried to do it amongst other things and messed it up!

```python
except toml.TomlDecodeError as err:
   logger.debug("Could not decode pyproject.toml: %s", err)
```

Should do the trick. In debug mode, it should tell you what the exception was sans trace. It *might be* enough to work out what is wrong with the toml file. I am not an expert there, so no sure how verbose the exceptions for it are.